pilot/docker/envoy_pilot.yaml.tmpl

```diff
@@ -2,7 +2,11 @@ admin:
   access_log_path: /dev/null
   address:
     socket_address:
+{{- if .IPv6Proxy }}
```

Comment:
How about passing {{ .LocalhostAddress }} ( either ::1 or 127.0.0.1 ) ? 

We want to simplify the templating, and seems more flexible.

Reply:
Ok, make sense, will do.